### PR TITLE
Toggle disabled when activate/deactive is processing

### PR DESCRIPTION
**Pull request by jakecraige**
screen shot 2015-03-10 at 10 06 28 am
Also disable clicking refresh repos multiple times

https://trello.com/c/6DGFfuoD/528-disable-activate-deactivate-button-in-repo-list-when-it-s-clicked-and-make-the-request

**Needs a designers touch** for the disabled state and the a became sized weird when changed to a button. Screenshot attached

### CODE REVIEW

File: app/views/templates/repo_list.haml [hunk 1]
```diff
@@ -2,7 +2,7 @@
   .search-wrapper
     %i.fa.fa-search
     %input.search{'type' => 'text', 'placeholder' => t('search_placeholder'), 'ng-model' => 'search.full_github_name'}
-  %a{'href' => '', 'class' => 'refresh', 'ng-click' => 'sync()', 'ng-class' => '{disabled: syncingRepos}'}
+  %button{'class' => 'refresh', 'ng-click' => 'sync()', 'ng-disabled' => 'syncingRepos'}
```
Comment (gylaz) [inline app/views/templates/repo_list.haml]: Thoughts on moving `refresh` out of params?
```haml
%button.refresh{...}
```

Comment (jakecraige) [inline app/views/templates/repo_list.haml]: I like it. Will do

**Comment by gylaz** [comment]
@frechg Do you mind doing the UI tweak here?

**Comment by frechg** [comment]
@gylaz @jakecraige just pushed a commit to fix up the button styles.

**Comment by salbertson** [comment]
Thanks @frechg!

@jakecraige, this looks good to me. :+1:

**Comment by jakecraige** [comment]
Merged in 0e4be20f233a2e303192bbf2daae9e54470d90a5 not sure why it didn't close

**Pull request closed without merging**